h/registerProduct.h

```diff
@@ -0,0 +1,37 @@
+
+
+const char *MODULE_REGISTER_USAGE = "IFAUSAGE";
+
+char productReg[COMMON_PATH_MAX];
+char productPID[COMMON_PATH_MAX];
+char productVer[COMMON_PATH_MAX];
+char productOwner[COMMON_PATH_MAX];
+char productName[COMMON_PATH_MAX];
+char productFeature[COMMON_PATH_MAX];
```

Comment:
Why do these have to be in a header? `zss.c` is the only user. There doesn't seem to be anything special about buffers.

Reply:
moved to zss.c